sources/platform/actors/development/actor_definition/source_code.md

```diff
@@ -9,13 +9,15 @@ sidebar_position: 4
 
 ---
 
-The Apify Actor's source code placement is defined by its [Dockerfile](./dockerfile.md). If you have created the Actor from one of Apify's [templates](https://apify.com/templates) then it's by convention placed in the `/src` directory.
+The structure and placement of an Apify Actor's source code is defined by its [`Dockerfile`](./dockerfile.md). If you create the Actor from one of Apify's [templates](https://apify.com/templates), the source code by default is placed in the `/src`.
 
-It's completely up to you what language and technologies, including various binaries (Chrome browser, Selenium, Cypress, or any other dependency of your choice) you use in your project. The only requirement is that you have to define the Dockerfile that will build the image for your Actor, including all the dependencies and your source code.
+You have the flexibility to choose any programming language, technologies, and dependencies (such as Chrome browser, Selenium, Cypress, or others) for your projects. The only requirement is to define a Dockerfile that builds the image for your Actor, including all dependencies and your source code.
 
 ## Example setup
 
-Let's take a look at the example JavaScript Actor's source code. The following Dockerfile
+Let's look at the example JavaScript Actor's source code structure.
+
+The following `Dockerfile`:
 
 ```dockerfile
 FROM apify/actor-node:16
```

Comment:
Let's update it to latest node js template with 
apify/actor-node:20

Reply:
Or maybe better remove version and just keep with 
apify/actor-node

Reply:
Based on proposed changes to dockerfile doc, I will put there `apify/actor-node:20` for consistency across docs